lib/cli/daemoncommand.cpp

```diff
@@ -191,6 +191,37 @@ pid_t l_UmbrellaPid = 0;
 static Atomic<bool> l_AllowedToWork (false);
 #endif /* _WIN32 */
 
+#ifdef I2_DEBUG
+/**
+ * Determine whether the developer wants to delay the worker process to attach a debugger to it.
+ *
+ * @return Internal.DebugWorkerDelay double
+ */
+static double GetDebugWorkerDelayCached()
+{
+	static double debugWorkerDelay = -1;
+
+	if (debugWorkerDelay != -1)
+		return debugWorkerDelay;
+
+	debugWorkerDelay = 0.0;
+
+	Namespace::Ptr internal = ScriptGlobal::Get("Internal", &Empty);
+
+	if (!internal)
+		return false;
```

Comment:
You're casting bool to double implicitly? What for?

Reply:
Thanks, oversight.